src/main/java/seedu/address/model/iteration/IterationDescription.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.model.iteration;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents an Iteration's description in the address book.
+ * Guarantees: immutable; any description is valid
+ */
+public class IterationDescription {
```

Comment:
Just an overall thought, since description, iterationdescription and feedback are all essentially the same maybe can use inheritance like description superclass? Or nested class? Else quite a lot of duplicate code.

Reply:
Ah yes, that's a good point. I think lots of areas in the code can actually be abstracted out. For instance, `additer` and `addcomm` actually share a lot of similar code. Perhaps we should discuss this further during our next team meeting and put this as a goal in our next version?